src/components/searchSection/myStyle.scss

```diff
@@ -0,0 +1,100 @@
+$gray-color: #979797;
+$pink-color: #eec197;
+$light-blue: #8bb4ea;
+$dark-blue: #7aa2d6;
+$light-gray: #f6f6f6;
+$light-blue2: #f9f9f9;
+$font-roboto: 'Roboto';
+$light-black: #0e0e0e;
+$light-pink: #fee4cb;
+.lightBlack {
+  background: $light-black;
+}
+
+.mycolorPink {
+  // background: $pink-color;
+  background: $light-pink;
+}
+.textPink {
+  color: $light-pink;
+}
+
+%fontType {
+  font-family: 'Saira';
+}
+
+.fontHeader1 {
+  @extend %fontType;
+  color: $dark-blue;
+}
+
+.fontHeader2 {
+  @extend %fontType;
+  color: $gray-color;
+  font-size: 1.1rem;
+}
+
+.parg1 {
+  color: $gray-color;
+  font-family: $font-roboto;
+  font-size: 0.9rem;
+}
+
+.mainContainer {
+  margin: 3rem 0 0 3rem;
+}
+
+.containerSearchSection {
+  width: 100%;
+  height: 35.3rem;
+}
+
+.searchBox {
+  height: 2.5rem;
+  position: relative;
+  top: -1.1rem;
+}
+.columnSearchBooks {
+  position: relative;
+  top: -1.4rem;
+  border-radius: 0.2rem;
+}
+
+.buttomText {
+  font-family: $font-roboto;
+  font-size: 1rem;
+}
+
+.buttomSearch {
+  width: 2.6rem;
+  height: 2.5rem;
+  border-radius: 5rem;
+}
```

Comment:
if this is  for the button so please fix the typo in the class name
also is this is the best way to do it? 
the border-radius is a bit corrupted, if you want to make circle the width and the height should be the same and the border-radius should 100%
Right?

Reply:
i remove it